README.md

```diff
@@ -41,24 +66,24 @@ and PYTHON_LIBS to avoid the need to call pkg-config.
 See the pkg-config man page for more details.
 ```
 
-### Short help (work around)
+### Quick fix (work around)
 
-Exec below:
+Execute the following:
 
 ```
-# Install python before install fontforge and ricty
+# Install python before installing fontforge and ricty
 brew install python
-# then install fontforge and ricty
+# Then install fontforge and ricty
 brew install ricty
 ```
 
 ### Long help (my environment)
 
-I use homebrew of course, but I don't use homebrew's python. I use pyenv via anyenv. In homebrew build script, your PKG_CONFIG_PATH are not used, there is homebrew's PKG_CONFIG_PATH. You can try this `brew sh` and `printenv | grep PKG_CONFIG_PATH`.
```

Comment:
I didn't really understand this part, so I left it mostly as it is and just reformatted a little bit.